index.js

```diff
@@ -0,0 +1,20 @@
+const express = require('express')
+const app = express()
+const bodyParser = require('body-parser')
+const { getMovieByTitle, getAllMovies, saveNewMovie } = require('./controllers/movie')
+
+
+app.get('/movies', getAllMovies)
+
+
+app.get('/movies/:title', getMovieByTitle)
+
+app.post('/', bodyParser.json(), saveNewMovie)
```

Comment:
Your post doesnt actually post to movies, this is posting to localhost:2000....
need `app.post('/movies', bodyParser.json(), saveNewMovie)`